### ISSUE TITLE
[Settings] Fonts to change? Three entries right now.

### ISSUE THREAD

**Issue by rubyFeedback** [geany/geany#3956]
Hi there,

I am using the latest git checkout source.

Under Fonts, in the settings, we have three entries:

1) Editor
2) Symbol List
3) Message Window

What I believe may be missing is the default fonts for the whole menu.

Perhaps the latter is done via the system theme, but I am using icewm, so ideally
I would like to be able to increase the other "missing" fonts as well. I have a very
wide monitor now and the default fonts are too small for me to easily see.

Changing the font size of e. g. the editor, works fine, so I have large fonts there,
but the e. g. at the menu "Interfaces", that part is small and I can not seem to be
able to increase or change the font size (in the settings widget).

Is it even possible to change the font there, from within geany? I assume geany
already has some way to query the font size, even if it comes from a theme, but
as said I use a simple WM, so I am not sure how to change the theme size. Ideally
we would be able to change ALL fonts, and if some are system-settings, perhaps
a simple small widget, available via some red alert icon, can indicate that the user
should not change these other fonts, unless the user knows what he or she is
doing (in addition to the current three one, which seem fine to change for everybody).

**Comment by elextr** [comment]
You are correct that the styling of most parts of Geany UI style is taken from the GTK theme so Geany "matches" the rest of the system.  If your system does not provide a GTK theme it will use a default fallback that GTK provides.  Anything in the style (not just font) you want to change for Geany only can be modified by GTK CSS in `Tools->Configuration Files->geany.css`.  

The only fonts that have GUI settings are those Geany widgets, eg the editor, symbol list, and message pane that are application specific rather than theme specific.
